_monitoring-plugins/alerting/security.md

```diff
@@ -82,3 +82,40 @@ If you only want users to be able to see and modify their own monitors and desti
 ```
 
 Then, use this new role for all alerting users. -->
+
+### Specify RBAC backend roles
+
+You can specify RBAC backend roles when you create or update a monitor with the Alerting API.
+
+In a create monitor scenario, follow these guidelines to specify roles:
+
+User type  | Role is specified by user or not (Y/N) | How to use the RBAC roles
+:--- | :--- | :---
+Admin user | User-specified | Use all the specified backend roles to associate to the monitor.
+Regular user | Yes | Use all the specified backend roles (that the user already has in their list of backend roles) to associate to the monitor.
```

Comment:
For `that the user already has in their list of backend roles`, should we say the list of backend roles that the user has permissions to?